Sources/ArgumentParser/Usage/DumpHelpGenerator.swift

```diff
@@ -9,11 +9,16 @@
 //
 //===----------------------------------------------------------------------===//
 
-@_implementationOnly import ArgumentParserToolInfo
-@_implementationOnly import class Foundation.JSONEncoder
+#if swift(>=5.11)
+private import ArgumentParserToolInfo
+private import class Foundation.JSONEncoder
+#else
+import ArgumentParserToolInfo
+import class Foundation.JSONEncoder
```

Comment:
I think @rauhul is right that `internal import` matches the existing behavior better than `private import`. Also, can we keep the `@_implementationOnly` attribute for older Swift versions, to maintain the separation?
```suggestion
internal import ArgumentParserToolInfo
internal import class Foundation.JSONEncoder
#else
@_implementationOnly import ArgumentParserToolInfo
@_implementationOnly import class Foundation.JSONEncoder
```

Reply:
`internal` sounds good to me. We can't keep `@_implementationOnly` for old versions and also solve the warning that lead to me submitting this. Since with Swift 5.10 it seems like it's an unfortunate in-between version where you can't have `internal import` yet, but you also can't avoid the warning for `@_implementationOnly` in all situations. I pushed a new version here which just carves that out so old versions still get `@_implementationOnly`. What do you think?